docs/data/charts/radar/radar.md

```diff
@@ -93,17 +87,48 @@ Otherwise, the other area would catch the pointer event, making it impossible to
 
 To remove highlight, set the `highlight` prop to `'none'`.
 
-## Tooltip 🚧
+## Tooltip
+
+Like other charts, the radar chart [tooltip](/x/react-charts/tooltip/) can be customized with slots.
+The `trigger` prop of the `tooltip` slot accepts the following values:
 
-## Composition 🚧
+- `'axis'`—the user's mouse position is associated with a metric. The tooltip displays data about all series along this specific metric.
+- `'item'`—when the user's mouse hovers over a radar area, the tooltip displays data about this series.
+- `'none'`—disable the tooltip.
+
+{{"demo": "RadarTooltip.js" }}
+
+## Composition
 
 For composition, use the `RadarDataProvider` to provide `series` and `radar` props.
 
-Providing components for radar composition is still a work in progress.
-If you miss some element or explanation, please open an issue describing what you want to achieve, and what is missing.
+In the background, you can render the `RadarGrid` and `RadarMetricLabels` that will display the grid and the labels.
 
-In this example, we uses `RadarSeriesArea` and `RadarSeriesMarks` to modify the order of the elements:
-all the marks are on top of all the path.
-Additionally, we apply different properties based on the series id.
+To display series, you have `RadarSeriesPlot` component (the area and marks).
+Or `RadarSeriesArea` and `RadarSeriesMarks` that allows to display at different level the area and marks.
+
+The `RadarAxisHighlight` component displays the axis highlight.
 
 {{"demo": "CompositionExample.js" }}
+
+For info here is the composition of the `RadarChart` component.
+
+```jsx
+<RadarDataProvider>
+  <ChartsWrapper>
+    <ChartsLegend />
+    <ChartsSurface>
+      // The background of the chart
```

Comment:
Not sure if we want to use this instead because using `//` comments isn't valid JSX:

```suggestion
      { /* The background of the chart */ }
```

Reply:
Yeah, I agree they are ugly, but if users copied and pasted that code it wouldn't work 😅